components/application-broker/internal/model.go

```diff
@@ -10,44 +11,83 @@ type ApplicationName string
 // ApplicationServiceID is an ID of Service defined in Application
 type ApplicationServiceID string
 
+type CompassMetadata struct {
+	ApplicationID string
+}
+
 // Application represents Application as defined by OSB API.
 type Application struct {
-	Name        ApplicationName
-	Description string
-	Services    []Service
+	Name                ApplicationName
+	Description         string
+	Services            []Service
+	CompassMetadata     CompassMetadata
+	DisplayName         string
+	ProviderDisplayName string
+	LongDescription     string
+	Labels              map[string]string
+	Tags                []string
+
+	// Deprecated, remove in #TBD-123
 	AccessLabel string
 }
 
 // Service represents service defined in the application which is mapped to service class in the service catalog.
 type Service struct {
-	ID                  ApplicationServiceID
-	Name                string
-	DisplayName         string
-	Description         string
-	LongDescription     string
+	ID                                   ApplicationServiceID
+	Name                                 string
+	DisplayName                          string
+	Description                          string
+	Entries                              []Entry
+	EventProvider                        bool
+	ServiceInstanceCreateParameterSchema map[string]interface{}
+
+	// Deprecated, remove in #TBD-123
+	LongDescription string
+	// Deprecated, remove in #TBD-123
 	ProviderDisplayName string
-
-	Tags   []string
+	// Deprecated, remove in #TBD-123
+	Tags []string
+	// Deprecated, remove in #TBD-123
 	Labels map[string]string
+}
 
-	//TODO(entry-simplification): this is an accepted simplification until
-	// explicit support of many APIEntry and EventEntry
-	APIEntry      *APIEntry
-	EventProvider bool
+func (s *Service) IsBindable() bool {
+	for _, e := range s.Entries {
+		if e.Type == "API" {
```

Comment:
Why don't you compare it with some const, e.g. `apiEntryType` from validator file? 
By the way, the same values (API, Events) are in both places `mapper.go` and `validator.go`.

Reply:
I've added proper const in `model.go` file and used them across all packages